rotate_test.go

```diff
@@ -1,4 +1,4 @@
-// +build linux
+// +build ignore
```

Comment:
🤔

Reply:
When running tests, rotate_test and example_test were getting pulled in, but these two files are really examples of how to use the package, rather than tests.

Reply:
Examples are run and tested on purpose -- they get shown in the godoc but also it ensures that it compiles correctly.  I think we leave this in, even though it's kinda dumb and leaks a goroutine and the associated state, but it's not otherwise bad.

Reply:
Turns out I didn't understand go examples.